### ISSUE TITLE
CFG misses blocks after function calls

### ISSUE THREAD

**Issue by gergo-** [angr/angr#475]
I have a small, statically linked ARM binary I would like to analyze, but angr's CFG is incomplete. The binary is at http://www.complang.tuwien.ac.at/gergo/krempel/buggy_binary

I am interested in the CFG of the function located at address `0x8000`. Here is the script I use to build the CFG and dump this function's blocks:
```
import angr
proj = angr.Project('buggy_binary')
cfg = proj.analyses.CFGAccurate(starts=[0x8000], normalize=True)
f = cfg.functions[0x8000]
for i, b in enumerate(list(f.blocks)):
    print 'block', i
    b.pp()
```

It identifies two blocks in the function:
```
block 0
0x8014:	mov	r0, r1
0x8018:	ldr	r1, [sp, #0xc]
0x801c:	bl	#0x8028
block 1
0x8000:	push	{r4, lr}
0x8004:	mov	r1, r0
0x8008:	mov	r0, #0x50
0x800c:	ldr	r4, [sp, #8]
0x8010:	bl	#0x8030
```

The problem is that the function's third block is missing. For reference, here is objdump's disassembly:

```
00008000 <fn1>:
    8000:	e92d4010 	push	{r4, lr}
    8004:	e1a01000 	mov	r1, r0
    8008:	e3a00050 	mov	r0, #80	; 0x50
    800c:	e59d4008 	ldr	r4, [sp, #8]
    8010:	eb000006 	bl	8030 <__aeabi_idivmod>
    8014:	e1a00001 	mov	r0, r1
    8018:	e59d100c 	ldr	r1, [sp, #12]
    801c:	eb000001 	bl	8028 <__aeabi_idiv>
    8020:	e0200004 	eor	r0, r0, r4
    8024:	e8bd8010 	pop	{r4, pc}
```

angr's CFG does not include the block with the last two instructions.

**Comment by ltfish** [comment]
Please try CFGFast (the default CFG) instead.

**Comment by ltfish** [comment]
But yeah, it should be a bug with `CFGAccurate`.

**Comment by ltfish** [comment]
Fixed in commit e737eb4. Thank you!
